_includes/pbc/all/install-features/202307.0/install-the-dynamic-store-prices-feature.md

```diff
@@ -0,0 +1,517 @@
+{% info_block warningBox %}
+
+Dynamic Multistore is currently running under an Early Access Release. Early Access Releases are subject to specific legal terms, they are unsupported and do not provide production-ready SLAs. They can also be deprecated without a General Availability Release. Nevertheless, we welcome feedback from early adopters on these cutting-edge, exploratory features.
+
+{% endinfo_block %}
+
+This document describes how to integrate the  Dynamic Store + Prices feature into a Spryker project.
+
+## Install feature core
+
+### Prerequisites
+
+To start feature integration, overview and install the necessary features:
+
+| NAME | VERSION |
+| --- | --- |
+| Spryker Core | {{page.version}} |
+| Prices | {{page.version}} |
+
+### 1) Set up configuration
+
+{% info_block warningBox "Configuration stores.php" %}
+
+Since the dynamic store is now enabled, configuration for the store is stored in the database, making the file `config/Shared/stores.php` deprecated. 
+
+The default store configuration will now be imported using new data import modules such as `CurrencyDataImport`. These modules will populate the store configuration in the database.
+A new major module `Currency`, have been introduced and that is responsible for extending store data and configuring it in the database. You can find migration guide [here](/docs/pbc/all/dynamic-multistore/{{page.version}}/base-shop/install-and-upgrade/update-the-currency-module.html)
+
+{% endinfo_block %}
+
+
+### 2) Set up the database schema and transfer objects
+
+1. Adjust the schema definition so entity changes trigger events:
+
+**src/Pyz/Zed/Currency/Persistence/Propel/Schema/spy_currency.schema.xml**
+
+```xml
+
+<?xml version="1.0"?>
+<database xmlns="spryker:schema-01" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="zed"
+          xsi:schemaLocation="spryker:schema-01 https://static.spryker.com/schema-01.xsd"
+          namespace="Orm\Zed\Currency\Persistence" package="src.Orm.Zed.Currency.Persistence">
+
+    <table name="spy_currency_store" idMethod="native">
+        <behavior name="event">
+            <parameter name="spy_currency_store_all" column="*"/>
+        </behavior>
+    </table>
+
+</database>
+
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure that the following changes have been applied by adding a currency to the store in the Backoffice interface. 
```

Comment:
1. We are usually verifying prior changes, not following. Should we move this note?
2. The verification is not clear. What outcome should I verify against?

Reply:
true. We can move this block in the end of the section, because it will not work until all steps are done. And we can rephrase it to smth like, Check that you can add a currency to the store on the product editing page in the backoffice